Warn users about ns label warnings (#787)
submariner-io/submariner-website: src/content/operations/deployment/subctl/_index.en.md

<!-- markdownlint-disable line-length -->
| Flag                             | Description
|:---------------------------------|:----------------------------------------------------------------------------|
| `--namespace` `<string>`         | Namespace in which validation pods should be deployed. If not specified, the `submariner-operator` namespace is used
| `--validation-timeout` `<value>` | Timeout in seconds while validating the connection attempt
| `--verbose`                      | Produce verbose logs during validation
<!-- markdownlint-enable line-length -->

{{% notice warning %}}
In Kubernetes 1.23 and above, the Pod Security admission controller expects namespaces to have security labels. Without these, you will see
warnings in `subctl`'s output. This will not actually impact `subctl`, but you can avoid the warnings and ensure correct behavior by

> adding these labels to the `namespace`:

Reviewer: This sentence gives an impression that users have to manually apply these labels even to the default namespace (which is "submariner-operator"). Its better to clarify that the warning mainly applies when user explicitly provides a different namespace while running the command.

<!-- markdownlint-disable line-length -->
| Flag                             | Description
|:---------------------------------|:----------------------------------------------------------------------------|
| `--namespace` `<string>`         | Namespace in which validation pods should be deployed. If not specified, the `submariner-operator` namespace is used
| `--validation-timeout` `<value>` | Timeout in seconds while validating the connection attempt
| `--verbose`                      | Produce verbose logs during validation
<!-- markdownlint-enable line-length -->

{{% notice warning %}}

> In Kubernetes 1.23 and above, the Pod Security admission controller expects namespaces to have security labels. Without these, you will see

Reviewer 2: ```suggestion
Starting with Kubernetes 1.23, the Pod Security admission controller expects namespaces to have security labels. Without these, you will see
```

Reviewer 2: @dfarrell07 probably can make a more grammatically correct suggestion :)

Reviewer 3: This looks good to me :+1:

warnings in `subctl`'s output. This will not actually impact `subctl`, but you can avoid the warnings and ensure correct behavior by
adding these labels to the user specified `namespace`:

```bash
"pod-security.kubernetes.io/enforce": "privileged"
"pod-security.kubernetes.io/audit": "privileged"
"pod-security.kubernetes.io/warn": "privileged"
```

> The default `submariner-operator` namespace already has these labels added.

Reviewer 2: This renders poorly and should probably ber before the code block (generally code block cant really nest inside a notice block)
 
![image](https://user-images.githubusercontent.com/51078027/177271005-ebd400fb-fff5-4755-98d8-af36f8e9a643.png)

Reviewer 2: This still happens

> 

Reviewer 2: ```suggestion
```

<!-- markdownlint-disable line-length -->
| Flag                             | Description
|:---------------------------------|:----------------------------------------------------------------------------|
| `--namespace` `<string>`         | Namespace in which validation pods should be deployed. If not specified, the `submariner-operator` namespace is used
| `--validation-timeout` `<value>` | Timeout in seconds while validating the connection attempt
| `--verbose`                      | Produce verbose logs during validation
<!-- markdownlint-enable line-length -->

> {{% notice warning %}}

Reviewer 2: This whole thing should be under the general section
```
### `diagnose`
```

> Starting with Kubernetes 1.23, the Pod Security admission controller expects namespaces to have security labels. Without these, you will see
> warnings in `subctl`'s output. This will not actually impact `subctl`, but you can avoid the warnings and ensure correct behavior by adding
> these labels to the user specified `namespace`:

Reviewer 2: ```suggestion
Starting with Kubernetes 1.23, the Pod Security admission controller expects namespaces to have certain security labels.
Without these labels, when using the `--namespace` flag you might see warnings in `subctl`'s output (this has no detrimental impact on `subctl diagnose`).
In order to avoid the warnings and ensure correct behavior by adding these labels to the namespace:
```

Reviewer 3: > In order to avoid the warnings and ensure correct behavior by adding these labels to the namespace:

Suggestion-suggestion:

> Avoid the warnings and ensure correct behavior by adding these labels to the namespace:

```bash
"pod-security.kubernetes.io/enforce": "privileged"
"pod-security.kubernetes.io/audit":   "privileged"
"pod-security.kubernetes.io/warn":    "privileged"
```

> 
> The default `submariner-operator` namespace already has these labels added.

Reviewer 2: ```suggestion
```

| `--verbose`                      | Produce verbose logs during validation
<!-- markdownlint-enable line-length -->

{{% notice warning %}}
In Kubernetes 1.23 and above, the Pod Security admission controller expects namespaces to have security labels. Without these, you will see

> warnings in subctl's output. subctl should work fine, but you can avoid the warnings and ensure correct behavior by adding the below labels 

Reviewer 3: ```suggestion
warnings in `subctl`'s output. This will not actually impact `subctl`, but you can avoid the warnings and ensure correct behavior by adding these labels 
```

The binary `subctl` should be `code style` and ideally shouldn't' start a sentence. Please see the Docs Style Guide for details. https://submariner.io/development/website/style-guide/
